
# HG changeset patch
# User Andrew John Hughes <gnu_andrew@member.fsf.org>
# Date 1366807128 -3600
# Node ID e562523c5037d8c6097afa00cc2932a959037a61
# Parent  d4e0f297cf8fb7a4e457d5bef8f0290623cf8b05
Update CACAO to latest tip.

2013-04-24  Andrew John Hughes  <gnu.andrew@member.fsf.org>

	* patches/cacao/jsig.patch:
	Removed.
	* Makefile.am:
	(CACAO_VERSION): Bump to current tip.
	(CACAO_SHA256SUM): Likewise.
	(ICEDTEA_PATCHES): Drop unneeded jsig patch.
	* NEWS: Updated.

diff -r d4e0f297cf8f -r e562523c5037 Makefile.am
--- a/Makefile.am	Wed Apr 24 10:24:40 2013 +0100
+++ b/Makefile.am	Wed Apr 24 13:38:48 2013 +0100
@@ -18,8 +18,8 @@
 LANGTOOLS_SHA256SUM = c412b61b095154fee4c45dc133f2baca3100fecd48b742f80da49a52ec473b02
 OPENJDK_SHA256SUM = 44c3e4a130fe4b76c1ba977ae2251884cefa774b82a24c4415b64395aef9594c
 
-CACAO_VERSION = a567bcb7f589
-CACAO_SHA256SUM = d49f79debc131a5694cae6ab3ba2864e7f3249ee8d9dc09aae8afdd4dc6b09f9
+CACAO_VERSION = e215e36be9fc
+CACAO_SHA256SUM = 4966514c72ee7ed108b882d9b6e65c3adf8a8f9c2dccb029f971b3c8cb4870ab
 CACAO_BASE_URL = http://icedtea.classpath.org/download/drops/cacao
 CACAO_URL = $(CACAO_BASE_URL)/$(CACAO_VERSION).tar.gz
 CACAO_SRC_ZIP = cacao-$(CACAO_VERSION).tar.gz
@@ -266,7 +266,6 @@
 if BUILD_CACAO
 ICEDTEA_PATCHES += \
 	patches/cacao/launcher.patch \
-	patches/cacao/jsig.patch \
 	patches/cacao/memory.patch
 else
 if USING_CACAO
diff -r d4e0f297cf8f -r e562523c5037 patches/cacao/jsig.patch
--- a/patches/cacao/jsig.patch	Wed Apr 24 10:24:40 2013 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,18 +0,0 @@
-diff -Nru cacao.orig/Makefile.am cacao/Makefile.am
---- cacao.orig/cacao/src/cacao/Makefile.am	2008-08-04 17:51:28.000000000 +0100
-+++ cacao/cacao/src/cacao/Makefile.am	2010-05-11 10:29:35.000000000 +0100
-@@ -96,12 +96,12 @@
- 	$(mkdir_p) $(prefix)/jre/lib/$(JAVA_ARCH)/server
- 	$(LN_S) -f $(libdir)/libjvm.so $(prefix)/jre/lib/$(JAVA_ARCH)/server
- 	$(ECHO) $(ECHO_N) > $(prefix)/jre/lib/$(JAVA_ARCH)/server/Xusage.txt
--	$(ECHO) $(ECHO_N) > $(prefix)/jre/lib/$(JAVA_ARCH)/server/libjsig.so
-+	$(ECHO) $(ECHO_N) > $(prefix)/jre/lib/$(JAVA_ARCH)/libjsig.so
- 
- uninstall-local:
- 	rm -f $(prefix)/jre/lib/$(JAVA_ARCH)/server/libjvm.so
- 	rm -f $(prefix)/jre/lib/$(JAVA_ARCH)/server/Xusage.txt
--	rm -f $(prefix)/jre/lib/$(JAVA_ARCH)/server/libjsig.so
-+	rm -f $(prefix)/jre/lib/$(JAVA_ARCH)/libjsig.so
- endif
- 
- 

